spec.bs

```diff
@@ -3786,20 +3795,67 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|configIDL|)</dfn> m
     1. [=list/Append=] |serverIg| to |igMap|[|owner|].
     1. If |ig|'s [=interest group/Private Aggregation coordinator=] is not null, then [=map/set=]
       |igPAggCoordinatorMap|[(|owner|, |name|)] to it.
-  1. Let |result| be a new {{AdAuctionData}}.
+  1. Let |results| be a new [=list=] of [=auction data per seller results=].
   1. Let |requestId| be the [=string representation=] of a [=version 4 UUID=].
-  1. [=map/Set=] |result|["{{AdAuctionData/requestId}}"] to |requestId|.
-  1. Let (|requestBlob|, |context|) be the result of serializing |igMap| with |config| and
-     |igPAggCoordinatorMap|. The serialization method may follow that described in
-     [Section 2.2.4 of Bidding and Auction Services](https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#name-generating-a-request).
-  1. Set |result|["{{AdAuctionData/request}}"] to |requestBlob|.
+  1. [=list/For each=] |config| of |configs|:
+    1. Let |seller| be |config|'s [=auction data config/seller=].
+    1. If |config|'s [=auction data config/coordinator=] is not one of the [=implementation-defined=]
+       coordinators supported by this [=user agent=]:
+      1. Let |result| be a new [=auction data per seller result=] with the following [=struct/items=]:
+        : [=auction data per seller result/seller=]
+        :: |seller|
+        : [=auction data per seller result/error=]
+        :: "invalid coordinator"
+      1. [=list/Append=] |result| to |results|.
+      1. [=iteration/Continue=].
+    1. Let |keyInfo| be the result of [=looking up the server encryption key=]
+       with |config|'s [=auction data config/seller=] and |seller|'s [=auction data config/coordinator=].
+    1. If |keyInfo| is failure:
+          1. Let |result| be a new [=auction data per seller result=] with the following [=struct/items=]:

```

Comment:
wrong indention. The preview shows how it's rendered.

Reply:
Fixed. Huh, I wonder how that happens.